src/fidesops/schemas/saas/saas_config.py

```diff
@@ -192,7 +192,35 @@ class SaaSType(Enum):
     custom = "custom"
 
 
-class SaaSConfig(BaseModel):
+class SaaSConfigBase(BaseModel):
+    """
+    Used to store base info for a saas config
+    """
+
+    fides_key: FidesOpsKey
+    name: str
+    type: SaaSType
+
+    @property
+    def fides_key_prop(self) -> FidesOpsKey:
+        return self.fides_key
+
+    @property
+    def name_prop(self) -> str:
+        return self.name
```

Comment:
This is so that I can refer to these props from the child class (https://github.com/ethyca/fidesops/pull/904/files/2cbb5f87272afa3587f7ee1a25ccfdc46d73b8ba#diff-dbbcf121d7b27a36a9632ced4799e5a662b37c7346fd0c9c97a1ff62cd793f12R282). super() only returns bound methods of parent class, not only props. Unless you know of a better way to do this?

Reply:
Did you get errors with `self`? As an example:

```py
class Parent:
    def __init__(self):
        self.first_name = "Ima"
        self.last_name = "Person"

    def full_name(self):
        return f"{self.first_name} {self.last_name}"


class Child(Parent):
    def __init__(self):
        self.middle = "Little"

        super().__init__()


child = Child()
print(child.full_name())
```

Prints:

```console
Ima Person
```

Using `self` feels cleaner, but if it is causing you problems somewhere this isn't a blocker.

Reply:
I did have problems with `self` but I also didn't implement an `__init__` method on either class. Is that required?

Reply:
```
class SaaSConfigBase(BaseModel):
    """
    Used to store base info for a saas config
    """

    fides_key: FidesOpsKey
    name: str
    type: SaaSType

class SaaSConfig(SaaSConfigBase):
   . . . 
   return Dataset(
            name=self.name,   # this fails
            collections=collections,
            connection_key=self.fides_key,     # this fails
        )

```

Reply:
Yes, the `super().__init__()` should make them available.